{"name": "test_sign_up_invalid_phone_num2", "status": "passed", "description": "the test is verify, that user can't sign up with invalid phone number", "start": 1680535729621, "stop": 1680535730406, "uuid": "ce856762-2b6d-4154-8b1b-7d6b28c58592", "historyId": "3b1e7642cb70f5d9b866e5fb6b27472f", "testCaseId": "83d801c547d346e613bdec82c5721eea", "fullName": "tests.chromeTests.testSignUpModal.TestSignUpModal#test_sign_up_invalid_phone_num2", "labels": [{"name": "parentSuite", "value": "tests.chromeTests"}, {"name": "suite", "value": "testSignUpModal"}, {"name": "subSuite", "value": "TestSignUpModal"}, {"name": "host", "value": "krot"}, {"name": "thread", "value": "19772-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.chromeTests.testSignUpModal"}]}